vyper/types.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, members):
         self.members = copy.copy(members)
 
     def __eq__(self, other):
-        return other.__class__ == StructType and other.members == self.members
+        return other.__class__ == TupleType and other.members == self.members
```

Comment:
Was this a mistake in the code?

Reply:
Yes, as we discussed on the pt1 PR ;) basically a mistake in the code (was copied from StructType definition).